<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="coppressTab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QListWidget" name="fileList">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>341</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addFileBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>add file</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeFileBtn">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>250</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>remove file(s)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="packFileBtn">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>330</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>pack</string>
      </property>
     </widget>
     <widget class="QPushButton" name="compressFileBtn">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>330</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>compress</string>
      </property>
     </widget>
     <widget class="QListWidget" name="folderList">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>30</y>
        <width>371</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addFolderBtn">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>260</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>add folders</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeFolderBtn">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>260</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>remove folders</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="compressPasswordEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>330</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>setPassword here</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sha1checkBtn">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>480</y>
        <width>541</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>sha1 check with exists compress file</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sha1sumEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>420</y>
        <width>551</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>420</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>sha1</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="extractTab">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLineEdit" name="targetFileEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>481</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="openZipFileBtn">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>110</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>open file</string>
      </property>
     </widget>
     <widget class="QPushButton" name="unpackBtn">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>220</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>unzip</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="extractPasswordEdit">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>220</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>230</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>set password here</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>é¡µ</string>
     </attribute>
     <widget class="QLineEdit" name="localBackupFolderEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="targetBackupFolderEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>341</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="chooseLocalFolderBtn">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>80</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>choose local folder</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chooseBackupFolderBtn">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>160</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>choose backup foder</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="backupTimer">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>250</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>250</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>select every  backup time</string>
      </property>
     </widget>
     <widget class="QPushButton" name="restoreBtn">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>250</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>restore</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
